src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -81,26 +83,21 @@ internal sealed partial class HttpConnection : IDisposable
 
             _connectionClose = false;
             _disposed = false;
+
+            Trace("Connection created.");
         }
 
         public void Dispose()
         {
-//#if DEBUG
```

Comment:
It's currently failing sporadically, I believe because there are some tests that don't actually complete things and just let everything drop.